docs/developer-guide/syntax-reference.rst

```diff
@@ -1242,6 +1242,18 @@ Scalar functions
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | ROUND                  |  ``ROUND(col1)``                                                          | Round a value to the nearest BIGINT value.        |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| SPLIT                  |  ``SPLIT(col1, delimiter)``                                               | Splits a string into an array of substrings based |
```

Comment:
Perhaps as a follow-up PR we can also include TRIM/RTRIM/LTRIM and SUBSTRING UDFs!

Reply:
Yeah. We could do that in other patches. The SUBSTRING is already available in KSQL, but the trim functions can be useful.